src/main/java/dave/storage/Storage.java

```diff
@@ -0,0 +1,160 @@
+package dave.storage;
+
+import java.time.LocalDateTime;
+import java.time.format.DateTimeFormatter;
+import java.util.Scanner;
+import java.io.File;
+import java.io.FileWriter;
+import java.io.IOException;
+import java.util.ArrayList;
+import dave.task.Task;
+import dave.task.TaskList;
+import dave.task.Event;
+import dave.task.Deadline;
+import dave.task.Todo;
+import dave.exceptions.InvalidDescriptionException;
+import dave.exceptions.InvalidDateTimeFormatException;
+
+
+/**
+ * The Storage class handles the loading and saving of tasks to and from a file.
+ */
+public class Storage {
+    private static String filePath;
+
+    String horizontal = "__________________________________________________________";
```

Comment:
```suggestion
    private String HORIZONTAL = "__________________________________________________________";
```
1. Constant names must be all uppercase using underscore to separate words (aka SCREAMING_SNAKE_CASE).
2. Good to add access modifier unless package private is intended.